### PR TITLE
ref(images loaded): Removed Clippedbox

### PR DESCRIPTION
**Pull request by robinrendle**

### CODE REVIEW

File: src/sentry/static/sentry/app/components/events/interfaces/debugMeta/index.tsx [hunk 1]
```diff
@@ -22,16 +22,14 @@ import SearchBar from 'app/components/searchBar';
...
 import {IconWarning} from 'app/icons';
 import {Organization, Project, Event, Frame} from 'app/types';
 
 import DebugImage from './debugImage';
 import {getFileName} from './utils';
 
 const MIN_FILTER_LEN = 3;
-const DEFAULT_CLIP_HEIGHT = 560;
-const PANEL_MAX_HEIGHT = 600;
+const PANEL_MAX_HEIGHT = 500;
```
Comment (priscilawebdev) [inline src/sentry/static/sentry/app/components/events/interfaces/debugMeta/index.tsx:32]: should we set `400` to be the same size as the breadcrumb list? or increase the breadcrumbs list height?

File: src/sentry/static/sentry/app/components/events/interfaces/debugMeta/index.tsx [hunk 2]
```diff
@@ -74,7 +72,7 @@ class DebugMeta extends React.PureComponent<Props, State> {
     filteredImages: [],
     showUnused: false,
     showDetails: false,
-    clipHeight: DEFAULT_CLIP_HEIGHT,
+    clipHeight: PANEL_MAX_HEIGHT,
```
Comment (priscilawebdev) [inline src/sentry/static/sentry/app/components/events/interfaces/debugMeta/index.tsx:75]: you can remove clipHeight as the component is no longer using it

File: src/sentry/static/sentry/app/components/events/interfaces/debugMeta/index.tsx [hunk 3]
```diff
@@ -411,11 +396,7 @@ class DebugMeta extends React.PureComponent<Props, State> {
       >
         <DebugImagesPanel>
           {filteredImages.length > 0 ? (
-            <ClippedBox
-              clipHeight={clipHeight}
-              renderedHeight={panelBodyHeight}
-              onReveal={this.handleOnReveal}
-            >
+            <div>
```
Comment (priscilawebdev) [inline src/sentry/static/sentry/app/components/events/interfaces/debugMeta/index.tsx:399]: do we need this `div` here? maybe we can replace it with `React.Fragment`

File: src/sentry/static/sentry/app/components/events/interfaces/debugMeta/index.tsx [hunk 4]
```diff
@@ -439,8 +420,8 @@ class DebugMeta extends React.PureComponent<Props, State> {
 
 export default DebugMeta;
 
-const StyledList = styled(List)<{overflowHidden: boolean; height: number}>`
-  ${p => p.overflowHidden && 'overflow: hidden !important;'}
+const StyledList = styled(List)<{height: number}>`
+  overflow: scroll;
```
Comment (priscilawebdev) [inline src/sentry/static/sentry/app/components/events/interfaces/debugMeta/index.tsx:424]: you don't need to set it here. I believe the list component already sets it to `auto`

**Comment by matejminar** [comment]
If I recall correctly, it was introduced because people found it annoying that it would "hijack" their scroll when they were scrolling down the page. But now also breadcrumbs are scrollable inside so I don't know 🤷‍♂️

File: src/sentry/static/sentry/app/components/events/interfaces/debugMeta/index.tsx (hunk 1 above)
Comment (robinrendle) [reply src/sentry/static/sentry/app/components/events/interfaces/debugMeta/index.tsx:32]: Agreed, done and done.

**Comment by robinrendle** [comment]
@priscilawebdev Thanks for the review! Updated it with your feedback

File: src/sentry/static/sentry/app/components/events/interfaces/debugMeta/index.tsx (hunk 1 above)
Comment (priscilawebdev) [reply src/sentry/static/sentry/app/components/events/interfaces/debugMeta/index.tsx]: thanks! 🙏

**Review by priscilawebdev — APPROVED** [review]
it looks good to me 👏

**Review by matejminar — APPROVED** [review]

**Pull request merged**